docs/edgeql/operators/json.rst

```diff
@@ -65,3 +80,57 @@ The contents of JSON *objects* and *arrays* can also be accessed via ``[]``:
     ... })
     ... FILTER j['name'] = <json>'std::bool';
     {{name: 'std::bool', timestamp: '2018-10-18'}}
+
+The element access operator ``[]`` will raise an exception if the
+specified index is not valid or does not exist for the base JSON
+value. It is possible to handle invalid indexes without raising an
+exception with the help of :eql:func:`json_get`.
+
+
+JSON Array Slice
```

Comment:
"Slicing JSON Arrays" ?

Reply:
I'd also fold "Accessing JSON Array Elements" (now it's "JSON Element Reference") into this section.

Reply:
OK